storage/src/journal/fixed.rs

```diff
@@ -88,7 +88,7 @@ pub struct Journal<E: Storage + Metrics, A> {
     // - Indices are consecutive and without gaps.
     // - There will always be at least one blob in the map.
     // - The most recent blob will never be completely full, but it may be empty.
-    blobs: BTreeMap<u64, (Write<E::Blob>, u64)>,
+    blobs: BTreeMap<u64, Write<E::Blob>>,
```

Comment:
I assume blob.size() is no longer a system call?  (I had this "cached" value here to avoid that overhead.)

Reply:
That's correct. Its needs to maintained by `buffer::Write` anyways, so figure better to track in 1 place than in both.